site/static/docs/5.2/assets/img/vite.svg

```diff
@@ -0,0 +1,15 @@
+<svg width="410" height="404" viewBox="0 0 410 404" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I think `width`, `height` and `fill` are safe to drop, here. 🙂

Reply:
Good catch! I didn't even think to look at the content of the file :)